src/lib/slider/slider.spec.ts

```diff
@@ -884,81 +929,99 @@ class SliderWithValueSmallerThanMin { }
 })
 class SliderWithValueGreaterThanMax { }
 
+@Component({
+  template: `<md-slider (change)="onChange($event)"></md-slider>`
+})
+class SliderWithChangeHandler {
+  onChange() { }
+}
+
 /**
  * Dispatches a click event from an element.
  * Note: The mouse event truncates the position for the click.
- * @param element The element from which the event will be dispatched.
+ * @param trackElement The track element from which the event location will be calculated.
+ * @param containerElement The container element from which the event will be dispatched.
```

Comment:
Instead of always passing in both the track element and the root slider element, why not just say the argument is `sliderElement` and `querySelector` for the trackElement?